desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/OverflowingItem.scss

```diff
@@ -28,3 +30,16 @@
     display: contents;
   }
 }
+
+.hue-path-browser__overflowing-breadcrumb {
+  min-width: 4ch;
+}
+
+.hue-path-browser__overflowing-menu {
+  width: 100%;
+  text-align: left;
+}
+
+.hue-path-browser__overflowing-last-breadcrumb {
```

Comment:
you could possibly use the :last-of-type pseudo selector instead of creating an additional class and simply things